openedx/features/edly/utils.py

```diff
@@ -400,3 +404,18 @@ def clean_django_settings_override(django_settings_override):
 
     if validation_errors:
         raise ValidationError(validation_errors)
+
+
+def get_marketing_link(marketing_urls, name):
+    """
+    Returns the correct URL for a link to the marketing site
+    """
+    link_map = settings.MKTG_URL_LINK_MAP
```

Comment:
I don't think we need to add check based on link_map as MKTG_URL will always be available in any case.

Reply:
Done.